src/ucs/config/parser.c

```diff
@@ -379,6 +379,145 @@ int ucs_config_sprintf_time(char *buf, size_t max, void *src, const void *arg)
     return 1;
 }
 
+int ucs_config_sscanf_speed(const char *buf, void *dest, const void *arg)
```

Comment:
maybe parse it as double rather than size_t?
also let's rename 'speed' to 'bandwidth' (its MB/s, not KM/h ... )